node/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+authors = ["KILT <info@kilt.io>"]
+build = 'build.rs'
+edition = '2018'
+homepage = 'https://kilt.io'
+license = 'Unlicense'
+name = 'node-portablegabi'
+repository = 'https://github.com/KILTprotocol/portablegabi-node'
+version = '2.0.0-rc2'
+
+[[bin]]
+name = 'node-portablegabi'
+
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
+
+[build-dependencies]
+substrate-build-script-utils = '2.0.0-rc2'
+
+[dependencies]
+futures = '0.3.4'
+log = '0.4.8'
+structopt = '0.3.8'
+node-portablegabi-runtime = { path = '../runtime', version = '2.0.0-rc2' }
```

Comment:
You are right, we do not need the version. I will remove this.

Concerning the versioning, I thought it would be easier to align with Substrate for outsiders. This immediately shows which Substrate version is currently working with the Portablegabi node.